Detectors/ITSMFT/ITS/tracking/include/ITStracking/TimeFrame.h

```diff
@@ -244,6 +245,7 @@ class TimeFrame
   std::vector<float> mMSangles;
   std::vector<float> mPhiCuts;
   std::vector<float> mPositionResolution;
+  std::vector<int> mClusterSize;
```

Comment:
why this needs to be int?

Reply:
Done. I needed to add an `if` for the treatement of overflows

Reply:
Can we merge it now?